src/nemo_run/__main__.py

```diff
@@ -12,7 +12,11 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import warnings
 
-from nemo_run.cli.api import create_cli
+warnings.simplefilter(action="ignore", category=DeprecationWarning)
+warnings.simplefilter(action="ignore", category=FutureWarning)
```

Comment:
Can we make this configurable via an env var in case user wants to see the warnings?

Reply:
PTAL.